### PR TITLE
(2237) Display organisation name on admin dashboard

### PR DESCRIPTION
**Pull request by Gweaton**
# Changes in this PR

- Displays user's organisation (or BEIS for service owners) on dashboard

### Before

image

#### BEIS user

image

#### Organisation user

image

### CODE REVIEW

File: src/app.controller.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class AppController {
 
     return {
       organisation: getUserOrganisation(actingUser),
+      isBEISUser: actingUser.serviceOwner,
```
Comment (bryangaledxw) [inline src/app.controller.ts:22]: I think we already get this? The user is available to all templates via `globalLocals`, so you can do {% if user.serviceOwner %} directly

File: views/admin/dashboard.njk [hunk 2]
```diff
@@ -29,6 +29,7 @@
     </nav>
   </div>
   <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l">{{ organisation | t }}</span>
```
Comment (bryangaledxw) [inline views/admin/dashboard.njk:32]: I'm not a fan of passing something into the `t` filter that might not actually need translating, I'd be happier if the translation happened within `getUserOrganisation` in the case where the use is a service owner. If we ever want automated checking to see if unrecognised string IDs are being passed into the translator, this'll defeat that

I think this is fine for now, but would like a ticket to address

**Review by pezholio — APPROVED** [review]
LGTM - just one thought - do we want to make `isBeisUser` a global, so we can use it everywhere?

File: src/app.controller.ts (hunk 1 above)
Comment (bryangaledxw) [reply src/app.controller.ts:22]: Also, do we want to mix terminology in code? I feel like I'd rather keep to `serviceOwner`, and references to "BEIS" live in the translation file

**Review by bryangaledxw — APPROVED** [review]

File: src/app.controller.ts (hunk 1 above)
Comment (Gweaton) [reply src/app.controller.ts]: Ah, sorry I didn't realise we had this in the globals already - have removed this commit.

File: views/admin/dashboard.njk (hunk 2 above)
Comment (Gweaton) [reply views/admin/dashboard.njk:32]: Yeah, I had the same concerns with this, but this was more of a quick fix. I'll add a ticket to address this later.

**Pull request merged**